src/styles/blocks/button.scss

```diff
@@ -0,0 +1,27 @@
+.button {
+  width: 100%;
+  height: 56px;
+
+  background: #f98921;
+  border-radius: 8px;
+  border: none;
+
+  font-family: "'Manrope'", sans-serif;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 27px;
+  color: #fff;
+  cursor: pointer;
+
+  transition: all 0.3s;
+
+  &:hover {
+    color: #fff;
+    background: #1b2129;
+  }
+
+  &:active {
+    background: #010810;
+    color: #fff;
```

Comment:
The color property in the &:active state is redundant as it has the same value as in the base state. You can safely remove this line.